src/MIPSolver/MIPSolverCplex.cpp

```diff
@@ -628,7 +630,7 @@ void MIPSolverCplex::activateDiscreteVariables(bool activate)
 
             for(int i = 0; i < numberOfVariables; i++)
             {
-                if(variableTypes.at(i) == E_VariableType::Integer /*|| variableTypes.at(i) == E_VariableType::Semiinteger*/)
+                if(variableTypes.at(i) == E_VariableType::Integer || variableTypes.at(i) == E_VariableType::Semiinteger)
```

Comment:
The reason I hesitated here was that I wasn't sure what the IloConversion is doing here. Does it turn an semi-integer into a semi-continuous and back? Is that what one wants? Or does it convert it into a continuous variable, but then also ensures correct bounds; and will it convert it back to a semi-integer variable correctly?

Reply:
Yes, after a closer look I agree. I completely disabled the integer relaxation feature if semicont. or semiint. variables are present since I am not myself what will happen.